### ISSUE TITLE
k2pdfopt is missing tesseract data

### ISSUE THREAD

**Issue by symphorien** [NixOS/nixpkgs#96492]
**Describe the bug**
k2pdfopt is missing tesseract data so I imagine OCR of french does not work so well.

```
k2pdfopt v2.51a (w/MuPDF,DjVuLibre,OCR) (c) 1970, GPLv3, http://willus.com
    Compiled Jan  1 1970 with Gnu C v9.2.0 for Linux on x64.

Initializing OCR for 2 threads ..
4.1.1 [SSE+AVX]
    Tesseract data folder = ''
    Tesseract languages: eng [LSTM+Tess]
```

**To Reproduce**
```
k2pdfopt -x -gs -ui- -col 1 /tmp/foo.pdf
```

**Expected behavior**
Tesseract has french :)

**Additional context**
the tesseract derivation has options to enable only specific languages, and defaults to all languages.

**Notify maintainers**
@bosu @danielfullmer 

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.7.15, NixOS, 20.03.2778.20904118004 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.6`
 - channels(root): `"nixos-20.03.2778.20904118004, nixos-hardware, nixos-unstable-20.09pre237781.32b46dd897a"`
 - channels(symphorien): `"home-manager-20.03"`
 - channels(foo): `"home-manager-20.03"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: [ k2pdfopt ]
# a list of nixos modules affected by the problem
module:
```

**Comment by danielfullmer** [comment]
See https://github.com/NixOS/nixpkgs/pull/93631/commits/56f6a3453d5ab83dc91044f07356c2f66c2c41c2 added to the PR to update k2pdfopt to 2.53: https://github.com/NixOS/nixpkgs/pull/93631

Looks like we just need to set `TESSDATA_PREFIX`.  In order to use a different language, use (for example): `-ocrlang chi_tra`.
To see the list of available languages run `k2pdfopt -ocrlang ?`
See also: https://www.willus.com/k2pdfopt/help/ocr.shtml

**Comment by symphorien** [comment]
Closed by #93631
